src/hooks/useKeyboardNavigation.js

```diff
@@ -57,10 +116,74 @@ const useKeyboardNavigation = ({
     ].filter(Boolean);
   }, []);
 
+  const focusNextEmojiListGroup = (columnIndex = 0) => {
+    const currentEmojiGroup = getCurrentEmojiListGroup();
+    const nextEmojiGroup = currentEmojiGroup.nextSibling;
+
+    if (nextEmojiGroup) {
+      nextEmojiGroup.children[columnIndex].firstChild.focus();
+
+      const categoryName = nextEmojiGroup.getAttribute('data-name');
+      activateCategoryByName(categoryName);
+    }
+  };
+
+  const focusPrevEmojiListGroupOnLastItem = () => {
+    const currentEmojiGroup = getCurrentEmojiListGroup();
+    const prevEmojiGroup = currentEmojiGroup.previousSibling;
+
+    if (
+      prevEmojiGroup &&
+      prevEmojiGroup.lastChild &&
+      prevEmojiGroup.lastChild.firstChild
+    ) {
+      prevEmojiGroup.lastChild.firstChild.focus();
+      return true;
+    }
+
+    return false;
+  };
+
+  const focusPrevEmojiListGroup = (columnIndex, itemsPerRow) => {
+    const currentEmojiGroup = getCurrentEmojiListGroup();
+    const prevEmojiGroup = currentEmojiGroup.previousSibling;
+
+    if (prevEmojiGroup) {
+      const numOfItems = prevEmojiGroup.children.length;
+
+      let nextFocusIndex;
+
+      for (let i = numOfItems - 1; i >= 0; i--) {
+        if (i % itemsPerRow === columnIndex) {
+          nextFocusIndex = i;
+          break;
+        }
+      }
+
+      const isInLastRow =
+        nextFocusIndex >= numOfItems - (numOfItems % itemsPerRow) ||
+        numOfItems % itemsPerRow === 0;
+
+      if (!isInLastRow) nextFocusIndex = numOfItems - 1;
+
+      prevEmojiGroup.children[nextFocusIndex].firstChild.focus();
+
+      const categoryName = prevEmojiGroup.getAttribute('data-name');
```

Comment:
This calls for a `getEmojiGroupName` function that will be used anywhere else as well. Will prevent the implementations from drifting off.